docs/architectural-decision-records/005-frontend-a11y-toolset.md

```diff
@@ -19,7 +19,7 @@ Together these tools include linting, testing, and auditing loaded web pages for
 
 [eslint jsx-a11y][eslint-a11y] Provides React-friendly code linting which allows developers to find accessibility issues during local development.
 
-[pa11y][pa11y] and [pa11y-ci][pa11y-ci] - Accessibility test runner for end to end testing in the command line or Node.js. End to end testing can catch concerns that are not noticeable in linting or unit tests since tests run on browser.
+[cypress-axe][cypress-axe] and [axe-core][axe-copre] - Accessibility test runner for end to end testing in the command line or Node.js. End to end testing can catch concerns that are not noticeable in linting or unit tests since tests run on browser. Our previous choice [pa11y][pa11y] seems to not be maintained anymore and no longer works with our end to end testing.
 
 [jest-axe][jest-axe] - This tool provides additional jest matchers, like `.toHaveNoViolations`, to use in unit tests. This may be redundant when end to end accessibility testing is present.
```

Comment:
Also a request - could you please also clarify that `jest-axe`is removed from our tool set?  Reason is we aren't using it and it was removed from package.json I think awhile ago so its not in use.